master/buildbot/www/config.py

```diff
@@ -53,11 +72,20 @@ def renderIndex(self, request):
             config.update({"user": session.user_info})
         else:
             config.update({"user": {"anonymous": True}})
+
+        www = self.master.config.www
+        versions = []
+        versions += self.env_versions 
```

Comment:
even if this is not a lot of work for master, I would prefer if the dictionary would be calculated inside the reconfigResource.